addon/doc/fr/readme.md

```diff
@@ -0,0 +1,98 @@
+# Notepad++ Module complémentaire pour NVDA #
```

Comment:
@josephsl is this how the system expects translations?

Reply:
It is a question of setting the right flags, not much more, a translation system should be able and flexible enough to scope with pre-translated versions IHMO. If volunteering translators sacrifice hours to make a translation, so should the translation manager not hesitate to invest some minutes too.
By the way, why don't we use *.po files for the documentation as well? I've tested it and it works pretty good.
The difference to the current system is:
- One readme.md (English)
- scons pot creates nvda.pot from the source, converts the *.md file to *.html (in the /doc/En/ folder) and creates the readme.pot from the latter.

The translators can update their localization with those pot files.
The reverse process is straight forward: Create an html for each language from the readme.po file and take the English html as template for it.
The needed functions are included in the translate toolkit package and are "html2po" and "po2html".
I attach a sample readme.po file.
[readme.zip](https://github.com/derekriemer/nvda-notepadPlusPlus/files/1277945/readme.zip)

Reply:
I don't know. Joseph helped with the translations stuff.